editor/editor_interface.cpp

```diff
@@ -713,6 +713,14 @@ void EditorInterface::save_all_scenes() {
 	EditorNode::get_singleton()->save_all_scenes();
 }
 
+Error EditorInterface::close_scene() {
+	if (!get_edited_scene_root()) {
```

Comment:
This check makes it impossible to close empty scenes. It should fail only if there is only one scene tab, there is no root and the scene has no path (empty scenes will retain path when you delete root).

Reply:
Fixed!

Reply:
> the scene has no path

This last part is not handled, but it's not important.